src/sol_imgui/sol_imgui.h

```diff
@@ -51,6 +60,18 @@ inline bool BeginChild(const std::string& name, float sizeX, float sizeY, bool b
 {
     return ImGui::BeginChild(name.c_str(), {sizeX, sizeY}, border, flags);
 }
+
+// new
```

Comment:
i wouldnt personally care about marking the new stuff

Reply:
reasoning - itis sort of obvious from the history of the changes done tothe file from git itself, IF, i would only marked things which are now deprecated for better visualisation when updating the file later